src/main/java/com/bina/varsim/tools/simulation/VCF2diploid.java

```diff
@@ -441,11 +441,16 @@ private boolean addVariant(final byte[] maskedSequence, final Sequence reference
             //NOTE: only translocation with ACCEPT subtype is marked as Translocation
             //translocation with REJECT subject is marked as Deletion
             if (variantType == VariantType.Translocation) {
-                if (variant.getPos2(allele) <= variant.getEnd2(allele)) {
-                    insertions = allSequences.getSequence(variant.getChr2(allele)).subSeq(variant.getPos2(allele), variant.getEnd2(allele) + 1);
-                } else {
-                    insertions = allSequences.getSequence(variant.getChr2(allele)).revComp(variant.getEnd2(allele), variant.getPos2(allele) + 1);
-                }
+              if (Variant.TranslocationSubtype.ACCEPT.equals(variant.getTranslocationSubtype(allele))) {
```

Comment:
Can't you use `Variant.TranslocationSubtype.ACCEPT == variant.getTranslocationSubtype(allele)` here?